targets/instnace.mk

```diff
@@ -56,7 +56,7 @@ instance.stop: local.hosts.revert  ## Stops your instance on GCP, but doesn't de
 
 instance.create:   ## Creates an empty instance for you on GCP.
 	@echo -e "Creating your virtual machine on GCP...   ($(INSTANCE_NAME))"
-	@gcloud compute instances create $(INSTANCE_NAME) \
+	gcloud compute instances create $(INSTANCE_NAME) \
```

Comment:
You can toggle the `VERBOSITY` variable in your `.config` file to `VERBOSITY=debug` instead of ~`VERBOSITY=critical`~. Same thing applies for `SHELL_OUTPUT` variable.